src/web/layouts/StandardLayout.tsx

```diff
@@ -480,7 +487,7 @@ export const StandardLayout = ({
                             />
                         </div>
                     </GridItem>
-                    <GridItem area="lines">
+                    <GridItem data-print-layout='hide' area="lines">
```

Comment:
if the parent is hidden, wouldnt you not need to have data-print-layout='hide' in this child?